keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -415,26 +401,31 @@ final {
     }
 
     internal::IValueStoreReader::attributes_t GetValueAsAttributeVector(uint64_t state_value) const {
-      return value_store_reader_->GetValueAsAttributeVector(state_value);
+        assert(value_store_reader_);
```

Comment:
ok, so should we change the compile flags then? This is somewhat strange to me, allowing to load without value store and then crashing hard if you call this 'not allowed' method on it.